x-pack/test/alerting_api_integration/common/fixtures/plugins/alerts/server/alert_types.ts

```diff
@@ -251,19 +320,63 @@ export function defineAlertTypes(
     producer: 'alertsFixture',
     defaultActionGroupId: 'default',
     validate: {
-      params: schema.object({
-        param1: schema.string(),
-      }),
+      params: paramsSchema,
+    },
+    async executor() {},
+  };
+  return result;
+}
+
+function getPatternFiringAlertType() {
```

Comment:
eh, we'll have a merge conflict here with https://github.com/elastic/kibana/pull/68437/files#diff-753fd9a204535630ddf989cebd2a4a76 - I had to make the patternFiring alert a little more interesting, to handle multiple instances.

Reply:
Ah thanks for the heads up! If your PR merges soon, I'll wait until then as the changes I made aren't hard to redo.